x-pack/test/security_solution_endpoint/apps/integrations_feature_flag/endpoint_exceptions.ts

```diff
@@ -0,0 +1,245 @@
+  const retry = getService('retry');
+  const esClient = getService('es');
+  const supertest = getService('supertest');
+  const find = getService('find');
+  const unzipPromisify = promisify(unzip);
+  const comboBox = getService('comboBox');
+  const toasts = getService('toasts');
+
+  describe('Endpoint Exceptions', function () {
+    targetTags(this, ['@ess', '@serverless']);
+
+    this.timeout(10 * 60_000);
+
+    const clearPrefilledEntries = async () => {
+      const entriesContainer = await testSubjects.find('exceptionEntriesContainer');
+
+      let deleteButtons: WebElementWrapper[];
+      do {
+        deleteButtons = await testSubjects.findAllDescendant(
+          'builderItemEntryDeleteButton',
+          entriesContainer
+        );
+
+        await deleteButtons[0].click();
+      } while (deleteButtons.length > 1);
+    };
+
+    const openNewEndpointExceptionFlyout = async () => {
+      await testSubjects.click('timeline-context-menu-button');
+      await testSubjects.click('add-endpoint-exception-menu-item');
+      await testSubjects.existOrFail('addExceptionFlyout');
+
+      await retry.waitFor('entries should be loaded', () =>
+        testSubjects.exists('exceptionItemEntryContainer')
+      );
+    };
+
+    const setLastFieldsValue = async ({
+      testSubj,
+      value,
+    }: {
+      testSubj: string;
+      value: string;
+      optionSelector?: string;
+    }) => {
+      const fields = await find.allByCssSelector(`[data-test-subj="${testSubj}"]`);
+
+      const lastField = fields[fields.length - 1];
+      await lastField.click();
+
+      await retry.try(
+        async () => {
+          await comboBox.setElement(lastField, value);
+        },
+        async () => {
+          // If the above fails due to an option not existing, create the value custom instead
+          await comboBox.setFilterValue(lastField, value);
+          await pageObjects.common.pressEnterKey();
+        }
+      );
+    };
+
+    const setLastEntry = async ({
+      field,
+      operator,
+      value,
+    }: {
+      field: string;
+      operator: 'matches' | 'is';
+      value: string;
+    }) => {
+      await setLastFieldsValue({ testSubj: 'fieldAutocompleteComboBox', value: field });
+      await setLastFieldsValue({ testSubj: 'operatorAutocompleteComboBox', value: operator });
+      await setLastFieldsValue({
+        testSubj: operator === 'matches' ? 'valuesAutocompleteWildcard' : 'valuesAutocompleteMatch',
+        value,
+      });
+    };
+
+    const checkArtifact = (expectedArtifact: object) => {
+      return retry.tryForTime(120_000, async () => {
+        const artifacts = await endpointArtifactTestResources.getArtifactsFromUnifiedManifestSO();
+
+        const foundArtifactId = artifacts
+          .flatMap((artifact) => artifact.artifactIds)
+          .find((artifactId) => artifactId.startsWith('endpoint-exceptionlist-macos-v1'));
+
+        expect(foundArtifactId).to.not.be(undefined);
```

Comment:
This part was modified, compared to ../integrations/endpoint_exceptions.ts, to properly parse new SO.